site/content/blogs/release-v122.en.md

```diff
@@ -35,6 +35,58 @@ Copilot v1.22 brings several new features and improvements:
 ## IAM Role Permissions Boundary
 
 ## FIFO SNS/SQS
+To have strict message ordering and message deduplication for your publish-subscribe architecture, you can now use SNS FIFO Topics and SQS FIFO Queues.
+
+### You can configure your manifest to have SNS FIFO Topics for your services.
+
+You can specify `fifo: true` in your service manifest under `publish.topics` as shown below and Copilot will create a FIFO SNS topic for you.
+
+```yaml
+publish:
+  topics:
+    - name: mytopic
+      fifo: true
+```
+
+Alternatively, you can also specify the advanced FIFO SNS Topic configuration as:
+```yaml
+publish:
+  topics:
+    - name: mytopic
+      fifo:
+        content_based_deduplication: true
+```
+
```

Comment:
Can we add links to the manifest specification at the bottom of the SNS and SQS sections, for example:
```
 For the full specification of FIFO topics, see [manifest specification](../docs/manifest/lb-web-service.en.md#theFifoID😅)
 
  For the full specification of FIFO queues, see [manifest specification](../docs/manifest/worker-service.en.md#theFifoID😅)
 ```